{
    "timestamp": "2024-02-28 16:46:36.637027",
    "exception": "user forced crash",
    "traceback": "Traceback (most recent call last):\n  File \"/home/adrian/Documents/GitHub/wdhs-alevel-flight-sim/OpenPyFlySim/flight_sim.py\", line 467, in main\n    raise Exception(\"user forced crash\")\nException: user forced crash\n",
    "variables": "[array([231. ,   0.8, 450. ]), array([270.,   0.,   0.]), 125.0, 16332, [], []]",
    "usertext": "",
    "checksum": "7ed4be371b44b19c6674dc73d1dfb1da4ec180383f1567dd7c0519eecda8de62eabcd57e9eaa6dd20d93eb5f227e423c",
    "systemDetails": [
        "Linux",
        "adrian",
        "6.5.0-21-generic",
        "#21~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Feb  9 13:32:52 UTC 2",
        "x86_64"
    ],
    "logchecksum": "da9d1e3cb23822ecf7b96ebad37ed2de",
    "sent": 0
}